src/control/lib/support/log.go

```diff
@@ -47,14 +47,15 @@ const (
 )
 
 type CollectLogSubCmd struct {
-	StopOnError  bool   `short:"s" long:"stop-on-error" description:"Stop the collect-log command on very first error"`
-	TargetFolder string `short:"t" long:"target-folder" description:"Target Folder location where log will be copied"`
-	Archive      bool   `short:"z" long:"archive" description:"Archive the log/config files"`
-	ExtraLogsDir string `short:"c" long:"extra-logs-dir" description:"Collect the Logs from given directory"`
-	LogStartDate string `short:"D" long:"start-date" description:"Specify the start date, the day from log will be collected, Format: MM-DD"`
-	LogEndDate   string `short:"F" long:"end-date" description:"Specify the end date, the day till the log will be collected, Format: MM-DD"`
-	LogStartTime string `short:"S" long:"log-start-time" description:"Specify the log collection start time, Format: HH:MM:SS"`
-	LogEndTime   string `short:"E" long:"log-end-time" description:"Specify the log collection end time, Format: HH:MM:SS"`
+	StopOnError           bool   `short:"s" long:"stop-on-error" description:"Stop the collect-log command on very first error"`
+	TargetFolder          string `short:"t" long:"target-folder" description:"Target Folder location where log will be copied"`
+	Archive               bool   `short:"z" long:"archive" description:"Archive the log/config files"`
+	ExtraLogsDir          string `short:"c" long:"extra-logs-dir" description:"Collect the Logs from given directory"`
+	LogStartDate          string `short:"D" long:"start-date" description:"Specify the start date, the day from log will be collected, Format: MM-DD"`
+	LogEndDate            string `short:"F" long:"end-date" description:"Specify the end date, the day till the log will be collected, Format: MM-DD"`
+	LogStartTime          string `short:"S" long:"log-start-time" description:"Specify the log collection start time, Format: HH:MM:SS"`
+	LogEndTime            string `short:"E" long:"log-end-time" description:"Specify the log collection end time, Format: HH:MM:SS"`
+	ExtraArgsCollectRsync string `short:"C" long:"cloud-storage" description:"Specify the destination in cloud storage"`
```

Comment:
I still think that the flag name and description here are too specific to the single use case that you are envisioning. Since this could be used as a more general solution, I suggest a more general approach, e.g.
```suggestion
FileTransferExecArgs string `short:"T" long:"transfer-args" description:"Extra arguments for alternate file transfer tool"`
```